### PR TITLE
Menu_mobile_display

### PR DESCRIPTION
**Pull request by Kal2403**
1. Added menu display function
2. Update the desktop navigation
3. Added linter for HTML, CSS and JS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,16 +15,21 @@
...
-        </ul>
-        <a href="#menu" class="nav-menu"><i class="fa-solid fa-bars nav-icon"></i></a>
-        <a href="#" class="nav-menu nav-menu-second"><i class="fa-solid fa-x nav-icon"></i></a>
-      </div>
+      <button id="close" class="close-menu"><i class="fa-solid fa-x"></i></button>
+      <ul class="side-bar">
+        <li class="nav-item"><a class="nav-link" href="#portfolio">Portfolio</a></li>
+        <li class="nav-item"><a class="nav-link" href="#about">About</a></li>
+        <li class="nav-item"><a class="nav-link" href="#contact">Contact</a></li>
+      </ul>
```
Comment (Zuhaib042) [inline index.html:25]: - [ ] There is an issue with your design. One of the requirements is that when the user clicks on any mobile-menu link, the mobile menu should disappear. you can achieve that by grabbing this `side-bar` in javascript and then implementing a forEach on that, where you will place the addEventListener on each element of that forEach and remove this class `nav.classList.remove('visible');` in the click event.
requirement: 
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.

File: index.html [hunk 2]
```diff
@@ -15,16 +15,21 @@
...
-      <div class="nav-container">
-        <ul class="side-bar" id="menu">
-          <li><a href="#portfolio">Portfolio</a></li>
-          <li><a href="#about">About</a></li>
-          <li><a href="#">Contact</a></li>
-        </ul>
-        <a href="#menu" class="nav-menu"><i class="fa-solid fa-bars nav-icon"></i></a>
-        <a href="#" class="nav-menu nav-menu-second"><i class="fa-solid fa-x nav-icon"></i></a>
-      </div>
+      <button id="close" class="close-menu"><i class="fa-solid fa-x"></i></button>
```
Comment (Zuhaib042) [inline index.html:20]: - [ ] Kindly consider fixing the position of this close-menu button. it should be exactly identical to the figma design. you can fix it using margin property.
- [ ] Also kindly consider fixing the background color. the background color of the mobile-menu is a bit dull in your design. consider making it identical to the figma design.
your design
s

figma design
 
fd

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @Kal2403 @kamranasif506 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Professional Readme
✅ Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA
### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**